packages/default/scss/menu/_variables.scss

```diff
@@ -2,51 +2,149 @@
+/// The background gradient of hovered menu item.
+/// @group menu
 $kendo-menu-item-hover-gradient: null !default;
 
-$kendo-menu-item-expanded-bg: null !default;
-$kendo-menu-item-expanded-text: $kendo-component-text !default;
-$kendo-menu-item-expanded-border: null !default;
-$kendo-menu-item-expanded-gradient: null !default;
+/// The background of active menu item.
+/// @group menu
+$kendo-menu-item-active-bg: null !default;
+/// The text color of active menu item.
+/// @group menu
+$kendo-menu-item-active-text: $kendo-link-text !default;
+/// The border color of active menu item.
+/// @group menu
+$kendo-menu-item-active-border: null !default;
+/// The background gradient of active menu item.
+/// @group menu
+$kendo-menu-item-active-gradient: null !default;
 
+/// The base shadow of focused menu item.
+/// @group menu
 $kendo-menu-item-focus-shadow: $kendo-list-item-focus-shadow !default;
 
-$kendo-menu-separator-spacing: k-spacing(1) !default;
+/// The background of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-bg: null !default;
+/// The text color of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-text: $kendo-link-text !default;
+/// The border color of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-border: null !default;
+/// The background gradient of selected menu item.
+/// @group menu
+$kendo-menu-item-selected-gradient: null !default;
 
+/// The background of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-bg: $kendo-component-bg !default;
-$kendo-menu-scroll-button-text: $kendo-link-text !default;
-$kendo-menu-scroll-button-border: $kendo-component-border !default;
+/// The text color of the menu scroll buttons.
+/// @group menu
+$kendo-menu-scroll-button-text: null !default;
+/// The border color of the menu scroll buttons.
+/// @group menu
+$kendo-menu-scroll-button-border: null !default;
+/// The background gradient of the menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-gradient: null !default;
 
+/// The background of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-bg: null !default;
+/// The text color of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-text: $kendo-link-hover-text !default;
+/// The border color of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-border: null !default;
+/// The background gradient of hovered menu scroll buttons.
+/// @group menu
 $kendo-menu-scroll-button-hover-gradient: null !default;
 
 
+// Menu Separator
+
+/// The inline margin of the horizontal menu separator.
+/// @group menu
+$kendo-menu-separator-margin-inline: k-spacing(1);
+/// The block margin of the horizontal menu separator.
+/// @group menu
+$kendo-menu-separator-margin-block: k-spacing(1.5);
+/// The border color of  menu separator.
+/// @group menu
+$kendo-menu-separator-border: var(--kendo-color-border, #{$kendo-component-border}) !default;
```

Comment:
Should it be `if($kendo-enable-color-system, k-color(border), $kendo-component-border)`